youtube_dl/extractor/sbs.py

```diff
@@ -64,3 +64,35 @@ def _real_extract(self, url):
             'id': video_id,
             'url': smuggle_url(self._proto_relative_url(theplatform_url), {'force_smil_url': True}),
         }
+
+    def _video_id(self, url):
+        return self._match_id(url)
+
+
+class SBSNewsIE(SBSIE):
+    _VALID_URL = r'^https?://(?:www\.)?sbs\.com\.au/news/(?P<id>[0-9a-z-]+)$'
+
+    _TESTS = [{
+        'url': 'https://www.sbs.com.au/news/are-the-campaigns-working-voters-speak-out',
+        'only_matching': True,
+        'md5': '2b73ddcbb597f24a87167826c47398f8',
+        'info_dict': {
+            'id': 'Vznr2YGb83mF',
+            'ext': 'mp4',
+            'title': 'Are the campaigns cutting through?',
+            'description': 'md5:d41d8cd98f00b204e9800998ecf8427e',
+            'thumbnail': r're:http://.*\.jpg',
+            'duration': 146,
+            'timestamp': 1557552900,
+            'upload_date': '20190511',
+            'uploader': 'SBSC',
+        },
+    }, {
+        'url': 'https://www.sbs.com.au/news/sbs-world-news-bulletin-may-11',
+        'only_matching': True,
+    }]
+
+    def _video_id(self, url):
+        slug = self._match_id(url)
+        page_contents = self._download_webpage(url, slug)
+        return self._search_regex(r'id="video-(\d+)"', page_contents, 'video id')
```

Comment:
As already pointed out: you must delegate to `SBSIE` extractor not inherit from it.

Reply:
There's no obvious delegation pattern elsewhere in this repository for a new contributor to follow.  

In 5d6e557, I extract SBSBaseIE with common _real_extract(), which follows established BaseIE pattern and keeps the code readable. Is this acceptable to you, @dstftw?

Reply:
Not acceptable. Search `url_result`.

Reply:
I'm sorry, @dstftw, could you please explain to me _why_ this solution is not acceptable? Or perhaps explain how the 400+ search results for the `url_result` utility method is relevant to implementing a common `_real_extract()` method? I'm really not clear on what it is I need to do to get this PR into the shape you want for it to be merged.